.github/workflows/ci.yml

```
@@ -0,0 +1,28 @@
+name: Jest Test
+
+on:
+  push:
+    branches: ["master", "main", "develop", "feat/init-repo"]
+  pull_request:
+    branches: ["master", "main", "develop", "feat/init-repo"]
```

No need for additional branches.

```suggestion
  push:
    branches: ["main", "develop"]
  pull_request:
    branches: ["main", "develop"]
```